### ISSUE TITLE
Server: Billing

### ISSUE THREAD

**Issue by joeelliott** [chhatch/websocket_chat#6]
Billing Server to manage user database (lightdb?)

Logging in does a check, is username taken, if not, prompt to register, if taken, prompt for password

New Registry:

username: same as the one attempted
class: select a Warrior, Shaman/Cleric, Mage, Warlock 
email address for later

**Comment by chhatch** [comment]
Yeah, we need to get something setup to persist user data. What do you mean by "billing server" here?
